src/components/common/applicant-info-3.vue

```diff
@@ -434,6 +410,9 @@ export default class ApplicantInfo3 extends Vue {
   async nextAction () {
     this.setIsLoadingSubmission(true)
     this.validate()
+    if (this.getShowCorpNum === CorpNumRequests.COLIN) {
+      await this.validateCorpNum(this.getCorpNum)
```

Comment:
This is good but:
1. while waiting for this to validate, I can still edit the other fields on the page -- this needs to be prevented; you can maybe use the full-page spinner for this (search for "showSpinner" events)
2. it would be nice if the error text is cleared when validation begins, and then is set again after it completes (otherwise I can see the old error while it's still validating, which is incorrect)

Reply:
it is a good point to add full-page spinner.

Reply:
Since you ticket has the UX Assurance label, please check with Tracey or Yui on item 2. If they're OK with it then I'm OK with it.

Reply:
Yui agreed with removing error on focusing.